_posts/2019-07-10-quarkus-0-19-released.adoc

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: 'Quarkus 0.19.0 released'
+date: 2019-07-10
+tags: quarkus release
+author: gsmet
+---
+
+Quarkus 0.19.0 was just released and is now available on Maven Central.
+
+This new version brings a lot of new extensions, especially in the persistence area.
+
+[NOTE]
+====
+Quarkus 0.19.0 is still targeting GraalVM 19.0.2 as we discovered a bug in GraalVM 19.1.0 that affects Quarkus.
+
+The issue is already fixed but we are waiting for a 19.1.1 release.
+====
+
+== What's new?
+
+Here are the main highlights of this release.
+
+=== Hibernate ORM schema management improvements
+
+We had a few issues with automatic schema upgrades handled by Hibernate ORM.
+
+Things are now smoother and we significantly improved the documentation to explain TODO link[how to configure schema management in various situations].
+
+=== New extensions
+
+==== MongoDB extension
+
+I don't think presenting https://www.mongodb.com/[MongoDB] is necessary anymore as it's probably the most well known NoSQL datastore.
```

Comment:
Perhaps it would be better to say: `MongoDB needs no introduction....`

Reply:
Yup, done.

Reply:
:+1: